src/engine/instrument.h

```diff
@@ -94,6 +94,7 @@ enum DivInstrumentType: unsigned short {
   DIV_INS_GBA_MINMOD=61,
   DIV_INS_BIFURCATOR=62,
   DIV_INS_SID2=63, // coincidence!
+  DIV_INS_SID3=64,
```

Comment:
Use instrument ID 66. 64 and 65 are reserved for #2087

Reply:
You should reserve e.g. 66 for OPL4, or is it reserved already?

Reply:
OPL4 doesn't need a new instrument type.

OPL4 PCM and MultiPCM use the same instrument type.